src/core/controls/DropDownList.js

```diff
@@ -0,0 +1,432 @@
+                    container.click = this.selectDropDownElement.bind(this, itemText._text);
+                    container.mouseover = function() { //TODO add throttle. cant use mouseenter
+                        this.handleEvent(DropDownList.HOVER_CONTAINER, i);
+                    }.bind(this);
+
+                    container.mouseout = function() {
+                        this.handleEvent(DropDownList.NORMAL);
+                    }.bind(this);
+
+
+                    container.addChild(itemText);
+                    inner.addChild(container);
+                }
+            }.bind(this));
+
+
+            wrapper.addChild(border);
+            wrapper.addChild(inner);
+
+            this.addChild(wrapper);
+        }
+    }
+    this.updateDropDown = false;
+};
+
+
+/**
+ * Hover state: mouse pointer hovers over the VariantContainer
+ * (ignored on mobile)
+ *
+ * @property HOVER
+ * @static
+ * @final
+ * @type String
+ */
+DropDownList.HOVER_CONTAINER = 'hover_container';
+
+/**
+ * State: initial state
+ * (ignored on mobile)
+ *
+ * @property NORMAL
+ * @static
+ * @final
+ * @type String
+ */
+DropDownList.NORMAL = 'normal';
+
+/**
+ * State: clicked state
+ *
+ * @property NORMAL
+ * @static
+ * @final
+ * @type String
+ */
+DropDownList.CLICKED = 'clicked';
+
+
+/**
+ * names of possible states for a dropddown
+ *
+ * @property stateNames
+ * @static
+ * @final
+ * @type String
+ */
+DropDownList.stateNames = [
+    DropDownList.HOVER_CONTAINER,DropDownList.NORMAL,DropDownList.CLICKED
+];
+
+/**
+ * handleEvent
+ */
+DropDownList.prototype.handleEvent = function(type, option) {
+    this.handelSubscribedCallouts(type);
+
+    if(type === DropDownList.HOVER_CONTAINER){
+        if(typeof option === 'number'){
+            this.currentState = DropDownList.HOVER_CONTAINER;
+            this.hoveredElementIndex = option;
+            this.cleanChilds();
+            this.updateDropDown = true;
+            this.updateLabel = true;
+        }
+    }
+    if(type === DropDownList.NORMAL){
+        this.currentState = DropDownList.NORMAL;
+        this.hoveredElementIndex = null;
+        this.cleanChilds();
+        this.updateDropDown = true;
+        this.updateLabel = true;
+    }
+};
+
+/**
+ * override events handlers
+ */
+DropDownList.prototype.mousedown = function(event) {
+    event.stopPropagation();
+    this.handleEvent(DropDownList.CLICKED);
+};
+
+/**
+ * override events handlers
+ */
+DropDownList.prototype.touchstart = function(event) {
+    event.stopPropagation();
+    this.handleEvent(DropDownList.CLICKED);
+};
+
+
+/**
+ * show/hide DropDown
+ */
+DropDownList.prototype.toggleDropDown = function () {
+    this.showDropDown = !this.showDropDown;
+    this.cleanChilds();
+    this.updateDropDown = true;
+    this.updateLabel = true;
+};
+
+
+/**
+ * initiate DropDown
+ * add event on stage that close dropDown on click
+ */
+DropDownList.prototype.initiate = function () {
```

Comment:
Having a popup management in gown is something I am currently working on for the PickerList, so the events would be handled by that manager.